docs/source/updating_capabilities.md

```diff
@@ -0,0 +1,341 @@
+# Updating the capability level of a channel
+
+*Audience: network administrators, node administrators*
+
+If you're not familiar with capabilities, check out [Capabilities](./capabilities_concept.html)
+before proceeding, paying particular attention to the fact that **node versions
+must be upgrading to the required level for all affected nodes before enabling
+capabilities**.
+
+For information about any new capability levels in the latest release of Fabric,
+check out [Upgrading your components](./upgrading_your_components.html#Capabilities).
+
+Note: when we use the term “upgrade” in Hyperledger Fabric, we’re referring to changing
+the version of a component (for example, going from one version of a binary to
+the next version). The term “update,” on the other hand, refers not to versions
+but to configuration changes, such as updating a channel configuration or a
+deployment script. As there is no data migration, technically speaking, in Fabric,
+we will not use the term "migration" or "migrate" here.
+
+## Overview
+
+In this tutorial, we will show the process for updating capabilities in all of the
+parts of the configuration of both the ordering system channel and any application
+channels.
+
+Whether you will need to update every part of the configuration for all
+of your channels will depend on the contents of the latest release as well as
+your own use case. For more information, check out [Upgrading to the latest version of Fabric](./upgrade_to_newest_version.html).
+Note that it may be necessary to update to the newest capability levels before
+using the features in the latest release, and it is considered a best practice
+to always be at the latest binary versions and capability levels.
+
+Because updating the capability level of a channel involves the configuration
+update transaction process, we will be relying on our [Updating a channel configuration](./config_update.html)
+topic for many of the commands.
+
+As with any channel configuration update, updating capabilities is, at a high
+level, a three step process (for each channel):
+
+1. Get the latest channel config
+2. Create a modified channel config
+3. Create a config update transaction
+
+We will enable these capabilities in the following order:
+
+1. [Orderer system channel](#orderer-system-channel-capabilities)
+
+  a. Orderer group
```

Comment:
This lettered list did not compile in markdown. Maybe bullets?

Reply:
Ok I'll try this.